core/src/test/java/org/apache/calcite/tools/PlannerTest.java

```diff
@@ -246,6 +248,24 @@ private String toString(RelNode rel) {
     }
   }
 
```

Comment:
How about add a JIRA link like `testOrderByNonSelectColumn` ?

Reply:
Good idea. I have added a JIRA link in the new commit.